lib/streamaggr/streamaggr.go

```diff
@@ -938,6 +945,10 @@ func (a *aggregator) Push(tss []prompbmarshal.TimeSeries, matchIdxs []byte) {
 				// Skip NaN values
 				continue
 			}
+			if a.threshold != nil && !math.IsNaN(*a.threshold) && s.Value < *a.threshold {
```

Comment:
It would be nice to have a [counter](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/5307a0f40afc9f4cc03a4ebd347bc7f42589c6a2/lib/streamaggr/streamaggr.go#L618) for this kind of dropped samples like `vm_streamaggr_ignored_samples_total{reason="underThreshold"}` when there is `threshold` defined in the rule.

Reply:
thanks for your suggestion. I have already fixed it.